lib/vagrant/action/general/package.rb

```diff
@@ -72,6 +72,7 @@ def call(env)
           @env = env
 
           self.class.validate!(env["package.output"], env["package.directory"])
+          package_with_folder_path(fullpath) if output_inside_folder?(env)
```

Comment:
Is this variable different from `@fullpath`? And if not...I think it can be accessed within that function without being passed in as a parameter.

Reply:
Thanks! :) I agree. A bit over 80 lines way to go then:

```ruby
package_with_folder_path if env["package.output"].include?(File::SEPARATOR)
```